app/Http/Controllers/AuthController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\AuthRequest;
+use App\Http\Requests\RegisterRequest;
+use App\interfaces\Services\IAuthService;
```

Comment:
Folder names should be Uppercase [PSR-12](https://www.php-fig.org/psr/psr-12/)

Other thing is that by default in the Laravel ecosystem, would be cool to call it a "Contract"
```suggestion
use App\Contracts\Services\AuthServiceContract;
```

```suggestion
use App\Interfaces\Services\AuthService;
```

This is just a preference thing, so feel free to implement it or not.